src/connectors/menu/__tests__/connectMenu-test.ts

```diff
@@ -499,24 +507,23 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/menu/js/#co
             results: new SearchResults(helper.state, [
               createSingleSearchResponse({
                 hits: [],
-                facets: {
-                  brand: 300,
-                },
```

Comment:
why is this removed?

Reply:
because it needs to be nested { brand: {Apple: 300}}, but this test doesn't care about that